gui/resources.xml

```diff
@@ -328,6 +334,7 @@ Vocabulary
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Toggle whether restoring the brush from a shortcut key also restores the saved color.</property>
           <signal name="activate" handler="context_toggle_color_cb"/>
         </object>
+        <accelerator key="asciitilde"/>
```

Comment:
I see.. cause my keyboard (US) the key is on the same row as the top row number keys. ^^'